src/pypi_attestation_models/_cli.py

```diff
@@ -0,0 +1,302 @@
+            _logger.debug("Attestation saved for %s saved in %s", file_path, signature_path)
+
+            signature_path.write_text(attestation.model_dump_json())
+
+
+def _inspect(args: argparse.Namespace) -> None:
+    """Inspect attestations.
+
+    Warning: The information displayed from the attestations are not verified.
+    """
+    for file_path in args.files:
+        if not file_path.is_file():
+            _die(args, f"{file_path} is not a file.")
+
+        try:
+            attestation = Attestation.model_validate_json(file_path.read_text())
+        except ValidationError as validation_error:
+            _die(args, f"Invalid attestation ({file_path}): {validation_error}")
+
+        _logger.info(
+            "Warning: The information displayed below are not verified, they are only "
+            "displayed. Use the verify command to verify them."
+        )
+
+        _logger.info(f"File: {file_path}")
+        _logger.info(f"Version: {attestation.version}")
+
+        decoded_statement = json.loads(attestation.envelope.statement.decode())
+
+        _logger.info("Statement:")
+        _logger.info(f"\tType: {decoded_statement['_type']}")
+        _logger.info("\tSubject:")
+        for subject in decoded_statement["subject"]:
+            _logger.info(f"\t\t{subject['name']} (digest: {subject['digest']['sha256']})")
+
+        _logger.info(f"\tPredicate type: {decoded_statement['predicateType']}")
+        _logger.info(f"\tPredicate: {decoded_statement['predicate']}")
+
+        if args.dump_bytes:
+            _logger.info(f"Signature: {attestation.envelope.signature}")
+
+        # Verification Material
+        verification_material = attestation.verification_material
+
+        # Certificate
+        certificate = x509.load_der_x509_certificate(verification_material.certificate)
+        _logger.info("Certificate:")
+        _logger.info(f"\tSubject: {certificate.subject.rfc4514_string()}")
+        _logger.info(f"\tIssuer: {certificate.issuer.rfc4514_string()}")
+        _logger.info(f"\tValidity: {certificate.not_valid_after_utc}")
+
+        # Transparency Log
+        _logger.info(
+            f"Transparency Log ({len(verification_material.transparency_entries)} entries):"
+        )
+        for idx, entry in enumerate(verification_material.transparency_entries):
+            _logger.info(f"\tLog Index: {entry['logIndex']}")
+
+
+def _verify(args: argparse.Namespace) -> None:
+    """Verify the files passed as argument."""
+    if args.staging:
+        verifier = Verifier.staging()
+    else:
+        verifier = Verifier.production()
+
+    pol = policy.Identity(identity=args.identity)
+
+    for file_path in args.files:
+        if not file_path.is_file():
+            _die(args, f"{file_path} is not a file.")
+
+        attestation_path = Path(f"{file_path}.publish.attestation")
+        if not attestation_path.is_file():
+            _die(args, f"Missing attestation file for {file_path}")
+
```

Comment:
Same for these (fail early if any of the files do not exist, before starting to process)